Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -44,7 +44,7 @@ namespace QuantConnect.Brokerages.Zerodha
     /// Zerodha Brokerage implementation
     /// </summary>
     [BrokerageFactory(typeof(ZerodhaBrokerageFactory))]
-    public partial class ZerodhaBrokerage : Brokerage, IDataQueueHandler, IDataQueueUniverseProvider, IOptionChainProvider
+    public partial class ZerodhaBrokerage : Brokerage, IDataQueueHandler, IDataQueueUniverseProvider
```

Comment:
For this first release of the brokerage (with Equity support only), we might also want to remove the `IDataQueueUniverseProvider` implementation, as it's only used with Options and Futures

Reply:
Fixed